ui_framework/components/local_nav/_local_search.scss

```diff
@@ -53,6 +58,38 @@
 }
 
 /**
+ * 1. When it expands, the search input should lie on top of the content.
+ * 2. Expand as the user enters text.
+ * 3. Cap the height so that we can offer auto-completion suggestions below the input.
+ * 4. When the input is collapsed, we want text input to be truncated with an ellipsis.
+ * 5. Prevent textarea from being resized.
```

Comment:
not sure if you wanted to prevent from resizing textarea by height ... i can still expand it